pages/isolasi-mandiri.tsx

```diff
@@ -30,8 +30,8 @@ export default function IsomanPage(props: IsomanPageProps) {
           backButton={<BackButton href="/" />}
           breadcrumbs={[
             {
-              name: "Isoman",
-              href: "/isoman",
+              name: "Isolasi Mandiri",
+              href: "/isolasi-mandiri",
```

Comment:
This breadcrumb should have been tested in the unit tests, just like any other breadcrums in the other pages.